### ISSUE TITLE
Android version for DeepFilterNet

### ISSUE THREAD

**Issue by tingyang01** [Rikorose/DeepFilterNet#302]
I am building a DeepFilterNet Android Version.
But Not sure how I can build the Android version.

**Comment by nimiai** [comment]
> 我正在构建 DeepFilterNet 的 Android 版本。 但我不确定如何构建 Android 版本。

did you make ok?

**Comment by ErDong-Sir** [comment]
> > 我正在构建 DeepFilterNet 的 Android 版本。 但我不确定如何构建 Android 版本。
> 
> did you make ok?

yes, I built it successfully and applied it to the Android platform, but the real-time performance is not good enough. The model has not been quantified for the client side.

**Comment by nimiai** [comment]
你好，我是艾春玲。您的邮件已收到，我会尽快回复。

**Comment by nimiai** [comment]
> > > 我正在构建 DeepFilterNet 的 Android 版本。 但我不确定如何构建 Android 版本。
> > 
> > 
> > did you make ok?
> 
> yes, I built it successfully and applied it to the Android platform, but the real-time performance is not good enough. The model has not been quantified for the client side.

maybe you can use model provide by https://github.com/KaleyraVideo/DeepFilterNet/tree/main

**Comment by ErDong-Sir** [comment]
> > > > 我正在构建 DeepFilterNet 的 Android 版本。 但我不确定如何构建 Android 版本。
> > > 
> > > 
> > > did you make ok?
> > 
> > 
> > yes, I built it successfully and applied it to the Android platform, but the real-time performance is not good enough. The model has not been quantified for the client side.
> 
> maybe you can use model provide by https://github.com/KaleyraVideo/DeepFilterNet/tree/main
Thank you. I have tried other mobile versions of the model before, but the real-time performance of relying solely on CPU inference is not good enough on my platform. Maybe the processor performance of my platform (a75*2 a55*6) is too poor

**Comment by nimiai** [comment]
> > > > > 我正在构建 DeepFilterNet 的 Android 版本。 但我不确定如何构建 Android 版本。
> > > > 
> > > > 
> > > > did you make ok?
> > > 
> > > 
> > > yes, I built it successfully and applied it to the Android platform, but the real-time performance is not good enough. The model has not been quantified for the client side.
> > 
> > 
> > maybe you can use model provide by https://github.com/KaleyraVideo/DeepFilterNet/tree/main
> > Thank you. I have tried other mobile versions of the model before, but the real-time performance of relying solely on CPU inference is not good enough on my platform. Maybe the processor performance of my platform (a75_2 a55_6) is too poor

this model use ok on A76， about 4.3K DMIPS
